.travis.yml

```diff
@@ -1,33 +1,33 @@
 language: python
```

Comment:
@Kajol-Kumari Please remove this file changes.

Reply:
@RishabhJain2018 These changes are coming due to the pre-commit hook. I am unable to revert them.

Reply:
okay!